<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Calendar</class>
 <widget class="QWidget" name="Calendar">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>EconoMystique</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <selectedon>img/econoLogo2.png</selectedon>
   </iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">background: #f4f4ec;
color: black;
</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>40</y>
     <width>801</width>
     <height>84</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="leftMargin">
     <number>290</number>
    </property>
    <item>
     <widget class="QLabel" name="EventsCalendar">
      <property name="font">
       <font>
        <family>Sans Serif Collection</family>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Events Calendar</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>801</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background: #365b6d;</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="titleLabel_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>221</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Cambria</family>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: white;</string>
    </property>
    <property name="text">
     <string>EconoMystique</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="btnSales">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>0</y>
      <width>93</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: #365b6d;
    color: white;
    border-radius: 50px;
}
QPushButton:hover {
    background-color: #5b8ca4;
}
QPushButton:pressed {
    border-style: inset;
    background-color: #365b6d;
}
</string>
    </property>
    <property name="text">
     <string>Sales</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnInventory">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>0</y>
      <width>93</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: #365b6d;
    color: white;
    border-radius: 50px;
}
QPushButton:hover {
    background-color: #5b8ca4;
}
QPushButton:pressed {
    border-style: inset;
    background-color: #365b6d;
}
</string>
    </property>
    <property name="text">
     <string>Inventory</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnCalendar">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>0</y>
      <width>93</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: #365b6d;
    color: white;
    border-radius: 50px;
}
QPushButton:hover {
    background-color: #5b8ca4;
}
QPushButton:pressed {
    border-style: inset;
    background-color: #365b6d;
}
</string>
    </property>
    <property name="text">
     <string>Calendar</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnAccount">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>0</y>
      <width>51</width>
      <height>41</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: #365b6d;
    color: white;
    border-radius: 50px;
}
QPushButton:hover {
    background-color: #5b8ca4;
}
QPushButton:pressed {
    border-style: inset;
    background-color: #365b6d;
}
</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <selectedon>img/pfBtn.png</selectedon>
     </iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>40</width>
      <height>40</height>
     </size>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>140</y>
     <width>761</width>
     <height>421</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout" stretch="5,0">
    <property name="leftMargin">
     <number>5</number>
    </property>
    <property name="rightMargin">
     <number>5</number>
    </property>
    <item>
     <widget class="QCalendarWidget" name="calendarWidget"/>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLabel" name="eventLabel">
        <property name="font">
         <font>
          <family>Sans Serif Collection</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>                    EVENT</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QListView" name="eventListView"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
